### PR TITLE
[Fix] Improve eventsim performance.

### PR DESCRIPTION
**Pull request by paul-amonson**
Signed-off-by: Paul Amonson <paul.d.amonson@intel.com>

### CODE REVIEW

**Comment by nbwhite** [comment]
I did a test on my local box, and got the following results:

Eventsim time: 6.82 s (14663 events/s)
Provider time 9.04 s (11060 events/s)
e2e: 27.33 s (3659 events/s)

This is comparable, and in fact better, than the pre-resiliency and Shasta 1.2 performance.

File: monitoring/src/test/java/com/intel/dai/monitoring/EventSimRasEventProviderForeignBusTest.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ public void processRawStringDataBadEventType() throws Exception {
         transformer_.processRawStringData(badSample2, config_);
     }
 
-    @Test(expected = NetworkListenerProviderException.class)
+    @Test //(expected = NetworkListenerProviderException.class)
```
Comment (nbwhite) [inline monitoring/src/test/java/com/intel/dai/monitoring/EventSimRasEventProviderForeignBusTest.java:98]: Does this Exception no longer happen?

Comment (paul-amonson) [reply monitoring/src/test/java/com/intel/dai/monitoring/EventSimRasEventProviderForeignBusTest.java:98]: True, it works more like the provider for API 1.2. Should I remove the comment?

**Review by sindhu-106 — APPROVED** [review]
After review discussions, the code changes looks good to me.

**Review by yhuan47 — APPROVED** [review]

**Pull request merged**